examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -147,14 +156,14 @@ def _save_forcefield(filename, ff_params):
     # update the forcefield parameters for a few steps, each step informed by a single free energy calculation
 
     # compute and save the sequence of relative_transformation indices
-    num_epochs = int(np.ceil(num_parameter_updates / len(relative_transformations)))
-    np.random.seed(2021)
+    num_epochs = int(np.ceil(args.param_updates / len(relative_transformations)))
+    np.random.seed(args.seed)
```

Comment:
Nice! Good idea to make the random seed configurable as well (e.g. when we wish to run replicates that differ only in the random seed)